tests/e2e/controlplane/control_plane_test.go

```diff
@@ -52,6 +53,7 @@ var _ = Describe("Control Plane Installation", Ordered, func() {
 			extraArg = "--set=platform=openshift"
 		}
 
+		fmt.Println("go version is:: ", runtime.Version())
```

Comment:
[1735055049] Control Plane Test Suite - 159/159 specs go version is::  go1.23.4
•••••Creating the necessary namespaces before running the tests...
go version is:  go1.23.4